src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -167,6 +168,7 @@ public override async Task<string> Execute(OrchestrationContext innerContext, st
                 this.Config.GetIntputOutputTrace(serializedInput),
                 FunctionType.Entity,
                 isReplay: false);
+            status = "Running";
```

Comment:
Rather than hardcoded strings, consider using `OrchestrationRuntimeStatus.Running.ToString()`. You could also consider using `OrchestrationRuntimeStatus` as the type of variable (instead of `string`) and then call `.ToString()` at the very end. The result is exactly the same, but it will help future maintainers understand what the acceptable range of status values are.

Reply:
That's a good point, hard-coded strings are flimsy. Incorporated your feedback in my latest changes!